### PR TITLE
Search ts21 8 added dotenv

### PR DESCRIPTION
**Pull request by sophielyon18**
https://dfacademy.atlassian.net/browse/TS21-8?atlOrigin=eyJpIjoiNTkwNjRiNWEwNDk5NDdjYjgzNzMzZGM4YmNkYmFiYzEiLCJwIjoiaiJ9

### CODE REVIEW

File: cypress/integration/search/search.spec.js [hunk 1]
```diff
@@ -32,22 +31,20 @@ describe('visiting the search page', function(){
...
     })
   })
-  context('Checks names are displayed and are links',()=>{
-    beforeEach(()=>{
-      cy.intercept('http://localhost:8080/api/search', { fixture: 'search-test.json' })
-      cy.intercept('http://localhost:8080/api/profile/*', {fixture: 'search-test.json' })
+  context('Checks names are displayed and are links', () => {
+    beforeEach(() => {
+      cy.intercept(`${Cypress.env('url')}/api/search`, { fixture: 'search-test.json' })
+      cy.intercept(`${Cypress.env('url')}/api/profile/*`, {fixture: 'search-test.json' })
```
Comment (AntMousinho) [inline cypress/integration/search/search.spec.js:37]: Maybe intercept with our fixture before linking to a profile/page just so the final visual output in cypress isn't an error screen?
`cy.intercept('http://localhost:8080/api/profile/*', { fixture: 'profile-users.json' })`

**Review by AntMousinho — APPROVED** [review]
[Ticket](https://dfacademy.atlassian.net/browse/TS21-8)
Completed the first four requirements of this ticket, tests are all passing in my local environment.

File: cypress/integration/search/search.spec.js (hunk 1 above)
Comment (namcap1) [reply cypress/integration/search/search.spec.js:37]: will do

**Comment by dearshrewdwit** [comment]
All you need is another internal review and you can merge this 👍

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**